oap-server/server-configuration/configuration-apollo/pom.xml

```diff
@@ -54,6 +54,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why do we need to add a test dependency but no code(test code) change?

Reply:
> Why do we need to add a test dependency but no code(test code) change?

This dependency was brought by ElasticSearch HLRC in `library-client` (as runtime dependency), since ES HLRC is removed, we have to add it explicitly, the same for other modules dependencies.